File: app/src/main/java/uk/co/alt236/apkcompare/app/util/FileSizeFormatter.java
package uk.co.alt236.apkcompare.app.util;

import org.apache.commons.io.FileUtils;

public class FileSizeFormatter {

    private final boolean format;

    public FileSizeFormatter(final boolean format) {
        this.format = format;
    }

    public String format(final long size) {
        if (format) {
            return FileUtils.byteCountToDisplaySize(size);
        } else {
            return String.valueOf(size);
        }
    }
}

File: app/src/main/java/uk/co/alt236/apkcompare/app/util/FileSizeFormatter.kt
package uk.co.alt236.apkcompare.app.util

import org.apache.commons.io.FileUtils

class FileSizeFormatter(private val format: Boolean) {

    fun format(size: Long): String {
        return if (format) {
            FileUtils.byteCountToDisplaySize(size)
        } else {
            size.toString()
        }
    }
}
